eoscostarica/eosio-dashboard
# Feat/update bp info with standard 863

### Update the displayed data of nodes and block producers according to bp json standard
 
### What does this PR do?

- Resolves #863
- Resolves #836 
- Resolves #874
- Refactor related components
- Hide "More Info" button on desktop

### Steps to test

1. Run the project locally
2. Check the paths of block producers and nodes

#### CheckList
